src/commands/networks/removeNetwork.ts

```diff
@@ -13,11 +14,15 @@ export async function removeNetwork(context: IActionContext, node?: NetworkTreeI
     nodes = await multiSelectNodes(
         { ...context, suppressCreatePick: true, noItemFoundErrorMessage: 'No networks are available to remove' },
         ext.networksTree,
-        NetworkTreeItem.contextValue,
+        NetworkTreeItem.customNetworkRegExp,
         node,
         nodes
     );
-
+    if (nodes.some(n => builtInNetworks.includes(n.networkName))) {
```

Comment:
The call to `multiSelectNodes` with `NetworkTreeItem.customNetworkRegExp` might automatically filter off all the built-in networks, making this redundant. I'll investigate.